update-docker-tags.go

```diff
@@ -79,6 +87,9 @@ Examples:
 	if err != nil {
 		log.Fatalf("failed to parse raw enforce, err: %s", err)
 	}
+	if dockerUsername == "" && dockerPassword == "" {
```

Comment:
```suggestion
	if dockerUsername == "" || dockerPassword == "" {
```

It won't work with just one of the two right?

Reply:
How about

https://github.com/sourcegraph/update-docker-tags/pull/3/files#diff-0c8db2e64ca607b73b274b4bbdff0aebe6d9dcd7c7cb4728de1dda381c9593ccR90-R95